variables.tf

```diff
@@ -87,6 +87,12 @@ variable "backup_retention_days" {
   default     = 35
 }
 
+variable "geo_redundant_backup_enabled" {
+  description = " - (Optional) Is Geo-Redundant backup enabled on the PostgreSQL Flexible Server. Defaults to false. Changing this forces a new PostgreSQL Flexible Server to be created."
```

Comment:
why does description start with dash as is it were a list?

Reply:
Copy paste of official doc